[Bug]: panic error when running grafana

### Problem description

When I try to run any grafana commands this error happens.
```$ grafana
panic: pattern "GET /debug/pprof/" (registered at /home/runner/.termux-build/_cache/go1.23.1-r1/src/net/http/pprof/pprof.go:100) conflicts with pattern "/debug/pprof/delta_heap" (registered at /home/runner/go/pkg/mod/github.com/grafana/pyroscope-go/godeltaprof@v0.1.7/http/pprof/pprof.go:24):
        GET /debug/pprof/ matches fewer methods than /debug/pprof/delta_heap, but has a more general path pattern

goroutine 1 [running]:
net/http.(*ServeMux).register(...)
        /home/runner/.termux-build/_cache/go1.23.1-r1/src/net/http/server.go:2797
net/http.HandleFunc({0x4001e14138?, 0x60a80a9217?}, 0x60ae7790a0?)
        /home/runner/.termux-build/_cache/go1.23.1-r1/src/net/http/server.go:2791 +0x9c
net/http/pprof.init.0()
        /home/runner/.termux-build/_cache/go1.23.1-r1/src/net/http/pprof/pprof.go:100 +0x94
```

### What steps will reproduce the bug?

Just running `grafana` will do. `1:11.1.3` was the last working version. Both `1:11.2.0` and `1:11.2.1` have the issue.

### What is the expected behavior?

If working, grafana will display some help text when just running `grafana`
```
$ grafana
NAME:
   grafana - Grafana server and command line interface

USAGE:
   grafana [global options] command [command options] 

VERSION:
   11.1.1

AUTHOR:
   Grafana Project <hello@grafana.com>

COMMANDS:
   cli        run the grafana cli
   server     run the grafana server
   apiserver  run a standalone api service (experimental)
   help, h    Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --help, -h     show help
   --version, -v  print the version
```

### System information

```shell
Termux Variables:
TERMUX_API_VERSION=0.49
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=13218
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.1
TERMUX__USER_ID=0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb file:///data/data/com.termux/files/home/drive/aptly/public/main/latest main_latest main
deb file:///data/data/com.termux/files/home/drive/aptly/public/main/today main_today main
deb file:///data/data/com.termux/files/home/drive/aptly/public/main/1_days main_1_days main
deb file:///data/data/com.termux/files/home/drive/aptly/public/main/3_days main_3_days main
deb file:///data/data/com.termux/files/home/drive/aptly/public/main/7_days main_7_days main
deb file:///data/data/com.termux/files/home/drive/aptly/public/main/30_days main_30_days main
deb file:///data/data/com.termux/files/home/drive/aptly/public/root/latest root_latest stable
deb file:///data/data/com.termux/files/home/drive/aptly/public/root/today root_today stable
deb file:///data/data/com.termux/files/home/drive/aptly/public/root/1_days root_1_days stable
deb file:///data/data/com.termux/files/home/drive/aptly/public/root/3_days root_3_days stable
deb file:///data/data/com.termux/files/home/drive/aptly/public/root/7_days root_7_days stable
deb file:///data/data/com.termux/files/home/drive/aptly/public/root/30_days root_30_days stable
deb file:///data/data/com.termux/files/home/drive/aptly/public/x11/latest x11_latest main
deb file:///data/data/com.termux/files/home/drive/aptly/public/x11/today x11_today main
deb file:///data/data/com.termux/files/home/drive/aptly/public/x11/1_days x11_1_days main
deb file:///data/data/com.termux/files/home/drive/aptly/public/x11/3_days x11_3_days main
deb file:///data/data/com.termux/files/home/drive/aptly/public/x11/7_days x11_7_days main
deb file:///data/data/com.termux/files/home/drive/aptly/public/x11/30_days x11_30_days main
Updatable packages:
All packages up to date
termux-tools version:
1.43.5
Android version:
14
Kernel build information:
Linux localhost 5.4.280-qgki-g004cf7a2803b #1 SMP PREEMPT Tue Sep 24 11:52:49 UTC 2024 aarch64 Android
Device manufacturer:
Fairphone
Device model:
FP5
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
Installed termux plugins:
com.termux.window versionCode:15
com.termux.api versionCode:49
com.termux.styling versionCode:1000
com.termux.widget versionCode:13
com.termux.boot versionCode:1000
```
```

--- a/packages/grafana/build.sh
+++ b/packages/grafana/build.sh
@@ -3,6 +3,9 @@ TERMUX_PKG_DESCRIPTION="The open-source platform for monitoring and observabilit
 TERMUX_PKG_LICENSE="AGPL-V3"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1:11.2.1"
+TERMUX_PKG_REVISION=1
+# Until https://github.com/grafana/grafana/pull/92055 is released:
+TERMUX_PKG_GO_USE_OLDER=true
 TERMUX_PKG_SRCURL=git+https://github.com/grafana/grafana
 TERMUX_PKG_BUILD_DEPENDS="yarn"
 TERMUX_PKG_BUILD_IN_SRC=true

